build.sh

```diff
@@ -112,6 +114,49 @@ build_managed_wcf()
     echo Build Exit Code = $BUILDERRORLEVEL
 }
 
+build_native_corefx()
```

Comment:
I'm not sure I agree with this, however I may be missing something. In this case it looks to me like this function is only called if "if $__buildnative;" evaluates to true (see line 396). Our WCF builds should set $__buildnative to false so we shouldn't have to worry about these native code paths and our diffs become much cleaner.